src/InstrumentationEngine.Lib/Logging.cpp

```diff
@@ -117,6 +137,14 @@ HRESULT CLogging::SetLoggingFlags(_In_ LoggingFlags loggingFlags)
     return s_loggerService.Get()->SetLoggingFlags(loggingFlags);
 }
 
+// static
+HRESULT CLogging::SetInstruMethodLoggingFlags(_In_ LoggingFlags loggingFlags)
```

Comment:
>InstruMethod [](start = 21, length = 12)

Is "InstruMethod" used elsewhere in the code? Seems a little awkward.

Reply:
It's used mostly in tests. I'll expand this.

---
In reply to: [345312339](https://github.com/microsoft/CLRInstrumentationEngine/pull/165#discussion_r345312339) [](ancestors = 345312339)